packages/app/src/debug/DebugContainer.cy.tsx

```diff
@@ -83,69 +83,81 @@ describe('<DebugContainer />', () => {
   })
 
   describe('testing util function: debugMapping', () => {
-    const createSpecs = (idArr: string[]) => {
-      const acc: {id: string}[] = []
-
-      idArr.forEach((val) => {
-        acc.push({ id: val })
-      })
-
-      return acc
-    }
-
     it('maps correctly for a single spec', () => {
-      const spec = createSpecs(['a1c'])
+      const specs = [
+        { id: 'a1c', groupIds: ['a'] },
+      ] as DebugSpecListSpecFragment[]
       const tests = [
         { specId: 'a1c', id: 'random1' },
         { specId: 'a1c', id: 'random2' },
-      ]
+      ] as DebugSpecListTestsFragment[]
+      const groups = [
+        { id: 'a', testingType: 'e2e' },
+        { id: 'b', testingType: 'e2e' },
+      ] as DebugSpecListGroupsFragment[]
 
-      const debugMappingArray = specsList(spec, tests)
+      const debugMappingArray = specsList({ specs, tests, groups, localSpecs: [], currentTestingType: 'e2e' })
```

Comment:
Now we are on Vite 4 you should have an easier time adding `vitest` to the code base. Not needed for this PR to merge, but I'd like to start using Vitest more for non UI tests - as much as I like dogfooding Cypress, it really doesn't feel like the right tool for this class of test. 

I know we are on the same page about this, just noting it here since we couldn't add Vitest before, but now we should be able to.